packages/terra-dropdown-button/src/SplitButton.jsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import KeyCode from 'keycode-js';
+import { injectIntl, intlShape } from 'react-intl';
+import DropdownButtonBase from './_DropdownButtonBase';
+import styles from './SplitButton.module.scss';
+import Button from './Button';
+
+const cx = classNames.bind(styles);
+
+const Variants = {
+  NEUTRAL: 'neutral',
+  GHOST: 'ghost',
+};
+
+const propTypes = {
+  /**
+   * The options to display in the dropdown. Should be comprised of the subcomponent `Button`.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Determines whether the primary button and expanding the dropdown should be disabled.
+   */
+  disabled: PropTypes.bool,
+  /**
+   * Determines whether the component should have block styles applied. The dropdown will match the component's width.
+   */
+  isBlock: PropTypes.bool,
+  /**
+   * Sets the text that will be shown on the primary button which is outside the dropdown.
+   */
+  primaryOptionLabel: PropTypes.string.isRequired,
+  /**
+   * What will be called when the primary button is pressed.
+   */
+  onClick: PropTypes.func.isRequired,
+  /**
+   * Sets the styles of the component.  The default is 'neutral'. Must be either 'neutral' or 'ghost'.
```

Comment:
I am a little confused here. The documentation says that a `SplitButton` could be of `ghost` variant but the [CSS is for `emphasis`](https://github.com/cerner/terra-core/blob/cb4c964627bac4eb94077e0647bd9a998708c394/packages/terra-dropdown-button/src/SplitButton.module.scss#L66-L68)? Also, it'd be nice to have an example on the site.

Reply:
I'm awaiting UX callouts for the ghost Dropdown Button